include/IECore/CamelCase.inl

```diff
@@ -41,6 +41,11 @@
 #include "boost/algorithm/string/classification.hpp"
 #include "boost/algorithm/string/split.hpp"
 
+// MSVC gets std::toupper from <cctype>
```

Comment:
This isn't unique to MSVC - we've just been lucky on Linux because `cctype` has been included coincidentally by other headers. Let's drop the `#ifdef`.

Reply:
Done and done.